client/src/pages/CreateOrganisationProfile.js

```diff
@@ -232,8 +233,10 @@ const CreateOrgProfile = (props) => {
               onChange={handleInputChangeEmail}
               style={styleInput}
               name="email"
+              className={errors.email && "has-error"}
```

Comment:
We already have a `style `attribute in `span ` element (line 251) which handles `errorStyles`. This line can be removed.

Reply:
@OzkanAbdullahoglu I have changed the code..can you please review

Reply:
Thank you, max chars limit increased to 50, this is coming from another discussion?

Reply:
Yes..but then another issue created for the email limit. This will not matter ..it will going to be changed